infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/Workflow.java

```diff
@@ -12,24 +12,37 @@
  * Created by Maithem on 10/25/17.
  */
 
-public interface Workflow {
+public abstract class Workflow {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "Workflow" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)